### ISSUE TITLE
Update to Nativescript 6

### ISSUE THREAD

**Issue by guix77** [NativeScript/nativescript-schematics#219]
**Environment**
```
{
  "name": "test",
  "nativescript": {
    "id": "org.nativescript.test"
  },
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "android": "tns run android --bundle",
    "ios": "tns run ios --bundle",
    "mobile": "tns run --bundle",
    "preview": "tns preview --bundle"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~7.2.0",
    "@angular/common": "~7.2.0",
    "@angular/compiler": "~7.2.0",
    "@angular/core": "~7.2.0",
    "@angular/forms": "~7.2.0",
    "@angular/http": "~7.2.0",
    "@angular/platform-browser": "~7.2.0",
    "@angular/platform-browser-dynamic": "~7.2.0",
    "@angular/router": "~7.2.0",
    "core-js": "^2.5.4",
    "nativescript-angular": "~7.2.0",
    "nativescript-theme-core": "~1.0.4",
    "reflect-metadata": "~0.1.12",
    "rxjs": "~6.3.3",
    "tns-core-modules": "~5.2.0",
    "zone.js": "^0.8.26"
  },
  "devDependencies": {
    "@angular/cli": "^7.2.0",
    "@angular/compiler-cli": "~7.2.0",
    "@angular-devkit/build-angular": "~0.11.4",
    "@nativescript/schematics": "~0.4.0",
    "@types/jasmine": "2.8.6",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "codelyzer": "~4.2.1",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~1.7.1",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.0",
    "karma-jasmine": "~1.1.1",
    "karma-jasmine-html-reporter": "^0.2.2",
    "nativescript-dev-typescript": "~0.8.0",
    "nativescript-dev-webpack": "^0.20.0",
    "protractor": "~5.3.0",
    "ts-node": "~5.0.1",
    "tslint": "~5.9.1",
    "typescript": "~3.1.1",
    "nativescript-dev-sass": "~1.6.0"
  }
}
```

**Describe the bug**

Using globally installed @angular/cli@8.1.2 and nativescript@6.0.2, I try to run:

`tns run android --bundle`

and I get:

```
$ tns run android --bundle
The current application is not compatible with NativeScript CLI 6.0.
Use the `tns migrate` command to migrate the app dependencies to a form compatible with NativeScript 6.0.
Running this command will not affect the codebase of the application and you might need to do additional changes manually – for more information, refer to the instructions in the following blog post: https://www.nativescript.org/blog/nativescript-6.0-application-migration
error Command failed with exit code 127.
```

I suppose this project needs an update for just released stable NativeScript 6.

**Comment by themounthead** [comment]
Adding to @guix77 , could the NS Team please look into this issue. I've upgrade to Angular 8 to be compatible with NativeScript CLI 6.0 but stuck with the schematics pointing to Angular 7.x and unable to make an Android build.

Appreciate your help. Thanks.

**Comment by kevincam3** [comment]
Same here

**Comment by overwise** [comment]
I'm totally stuck as well... I'm using Sidekick, which now insists on working with latest nativescript. Hope some migration will come...

**Comment by sis0k0** [comment]
Hey everyone! The team is actively working on updating the package.

**Comment by overwise** [comment]
Thanks, very nice to hear that! @sis0k0 if I'm not too pushy, is there any forecast about when to expect the update?

**Comment by sis0k0** [comment]
fixed with v0.7.0
